Readme.md

```diff
@@ -1,126 +1,58 @@
-Pest Formatter
-==============
-
-Try Online: [PlayGround](https://sbeckeriv.github.io/pest_format/)
```

Comment:
this is still live, but I guess it won't be updated -- or @sbeckeriv do you plan to maintain the online tool?

Reply:
After this PR merged, we can implement format on pest site by use WASM.